byterun/startup.c

```diff
@@ -192,7 +192,7 @@ static char * read_section(int fd, struct exec_trailer *trail, char *name)
 
 #ifdef _WIN32
 
-static wchar_t * read_section_to_utf16(int fd, struct exec_trailer *trail, char *name)
+static wchar_t * read_section_to_os(int fd, struct exec_trailer *trail, char *name)
```

Comment:
That's where I found wchar_t.  Should this be charnat for consistency?

Reply:
Yes.

Reply:
Ping on this minor point.

Reply:
Thanks, fixed.